content/post/2022-01-21-friday-hacks-214.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Friday Hacks #214, January 21"
```

Comment:
Oh and i notice past FH posts don't have the date in the title as well, since it is repeated in the date. Oh and the time starts at 6.30pm when i preview locally, will it be better if we say snacks will be served at 6.30pm and the first session starts at 7pm?

Reply:
Yes I think it would be better, I'll edit the message!